scst-store/amr/graphql-query.hbs.md

```diff
@@ -0,0 +1,181 @@
+  appAcceleratorRuns(query:{guid: "d2934b09-5d4c-45da-8eb1-e464f218454e"})
+  ```
+
+- `source`: string representing the client used to run the accelerator. Supported values include `TAP-GUI`, `VSCODE`, and `INTELLIJ`.
+
+  For example:
+
+  ```graphql
+  appAcceleratorRuns(query:{source: "TAP-GUI"})
+  ```
+
+- `username`: string representing the user name of the person who runs
+the accelerator, as captured by the client UI.
+  For example:
+
+  ```graphql
+  appAcceleratorRuns(query:{username: "homer.simpson"})
+  ```
+
+- `namespace` and `name`: strings representing the accelerator that
+was used to create an application.
+  For example:
+
+  ```graphql
+  appAcceleratorRuns(query:{name: "tanzu-java-web-app"})
+  ```
+
+- `appAcceleratorRepoURL`, `appAcceleratorRevision`, and `appAcceleratorSubpath`: actual location in VCS (Version Control System) about the accelerator sources used.
+  For example:
+
+  ```graphql
+  appAcceleratorRuns(query:{
+    appAcceleratorRepoURL: "https://github.com/vmware-tanzu/application-accelerator-samples.git",
+    appAcceleratorRevision: "v1.6"
+  })
+  ```
+
+- `timestamp`: string representation of the exact time the accelerator ran. You can query for runs that happened `before` or `after` a particular instant:
+  For example:
+
+  ```graphql
+  appAcceleratorRuns(query: {timestamp: {after: "2023-10-11T13:40:46.952Z"}})
+  ```
+
+### <a id='app-accelerator-runs-fields'></a> AppAcceleratorRuns fields
+
+You can choose the following fields to return in the GraphQL query.
+See the section above for details about those fields.
+You must specify at least one field.
+
+- `guid`: UID identifying the run
+- `source`: string representing the client used to run the accelerator
+- `username`: string representing the user name of the person who ran
+  the accelerator
+- `namespace` and `name`: strings representing the accelerator which
+  was used to create an application
+- `appAcceleratorRepoURL`, `appAcceleratorRevision`, and `appAcceleratorSubpath`: actual location in VCS of the sources of the
+  accelerator used
+- `appAcceleratorSource`: VCS information of the sources of the accelerator used, but navigable as a [Commit](./data-model-and-concepts.hbs.md#commits)
+- `timestamp`: the exact time the accelerator was run
+- `appAcceleratorFragments`: a one-to-many container of nodes representing the fragment versions used in each AppAcceleratorRun. Those fragment nodes share many of the fields with AppAcceleratorRun, with the same semantics but applied to the particular fragment. Those include:
+  - `namespace` and `name`: strings representing the identity of the fragment
+  - `appAcceleratorFragmentSourceRepoURL` , `appAcceleratorFragmentSourceRevision`, and  `appAcceleratorFragmentSourceSubpath`: actual location in VCS of the sources of the fragment used
+  - `appAcceleratorFragmentSource`: VCS information of the sources of the fragment, but navigable as a [Commit](./data-model-and-concepts.hbs.md#commits)
```

Comment:
This section is duplicating the content above, can it be removed?

Reply:
We can combine the two, but we'd need to somehow indicate to users which fields can be used as query arguments.